examples/SerialIface/vgm.py

```diff
@@ -42,6 +42,11 @@ def play(opl, vgm_stream):
       delay_us = _samples_to_us(delay_samples)
     elif opcode == END_DATA:
       break
+    elif opcode >= 0x51 and opcode <= 0x5f:
+      # opcodes for writing to other YMF chips, skip
+      vgm_stream.read(2)
+    elif opcode == 0:
```

Comment:
@vincentbernat What does this 0x0 mean? It isn't documented in any spec that I can see.
http://vgmrips.net/wiki/VGM_Specification
http://www.smspower.org/uploads/Music/vgmspec170.txt

Reply:
I don't know either. Remove the condition and try to play the `zero_wing.vgz`. It fails about an opcode 0. Looking at the source code of `opl2play.c` (for the Pi), I see this opcode is just ignored. Maybe this hides another bug.

Reply:
You're probably right, it may be masking another issue. Thanks for the information.